lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +200,41 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self._config = config
+        self.urls = []
+        self._seed_urls = self._config.get_seed_urls()
 
-    def _extract_url(self, article_bs: BeautifulSoup) -> str:
+    @staticmethod
+    def _extract_url(article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        url: Union[str, list, None] = article_bs.get('href')
```

Comment:
why do you need this line here?

Reply:
The "Union[str, list, None]" part of the code specifies that the variable "url" can be either a string, a list, or None